packages/web/src/components/gcds-link/gcds-link.tsx

```diff
@@ -0,0 +1,221 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Watch,
+  Prop,
+  State,
+  h,
+} from '@stencil/core';
+import { assignLanguage, observerConfig } from '../../utils/utils';
+import { inheritAttributes } from '../../utils/utils';
+import i18n from './i18n/i18n';
+
+@Component({
+  tag: 'gcds-link',
+  styleUrl: 'gcds-link.css',
+  shadow: true,
+})
+export class GcdsLink {
+  @Element() el: HTMLElement;
+
+  private shadowElement?: HTMLElement;
+
+  /**
+   * Props
+   */
+
+  /**
+   * Set the link size
+   */
+  @Prop({ mutable: true }) size: 'regular' | 'small' | 'inherit' = 'regular';
+
+  @Watch('size')
+  validateSize(newValue: string) {
+    const values = ['regular', 'small', 'inherit'];
+
+    if (!values.includes(newValue)) {
+      this.size = 'regular';
+    }
+  }
+
+  @Watch('display')
+  validateDisplay(newValue: string) {
+    const values = ['block', 'inline', 'inline-block'];
+
+    if (!values.includes(newValue)) {
+      this.display = 'block';
+    }
+  }
+
+  /**
+   * Link props
+   */
```

Comment:
Sorry super picky, but can we remove this comment? It was only needed in the button component because we had button and link-specific props so we can remove it here.